diff --git a/source/app/components/_common/services/router-helper.provider.js b/source/app/components/_common/services/router-helper.provider.js
index 426cf72..5319d32 100644
--- a/source/app/components/_common/services/router-helper.provider.js
+++ b/source/app/components/_common/services/router-helper.provider.js
@@ -101,7 +101,7 @@ class RouterHelperProvider {
             mainTitle: '',
             resolveAlways: {}
         };
-        this.$locationProvider.html5Mode(true);
+        this.$locationProvider.html5Mode(false);
     }
 
     configure (cfg) {
diff --git a/source/app/index.jade b/source/app/index.jade
index edbe21b..f16f9ab 100644
--- a/source/app/index.jade
+++ b/source/app/index.jade
@@ -10,7 +10,7 @@ html(lang="en", ng-app= app, ng-strict-di)
         meta(charset="utf-8")
         meta(http-equiv="X-UA-Compatible", content="IE=edge, chrome=1")
         meta(name="viewport", content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no")
-        base(href="/")
+        base(href="./")
         //- babel polyfill for IE
         //- conditional comment is not supported from IE 10
         //- TODO: how to include this only for IE
